eos/constraints/B-to-V-form-factors.yaml

```diff
@@ -324,73 +324,173 @@ B->D^(*)::FormFactors[parametric,LCSRLattice]@GKvD:2018A:
+      { form-factors: 'BGL1997' }, { form-factors: 'BGL1997' }, { form-factors: 'BGL1997' }  ]
   means:
-  - 1.0608964598622037
-  - 0.7227666600037261
-  - 0.9389948027037445
-  - 0.8536474175438934
-  - 1.2069222026563986
-  - 0.7576136507164594
-  - 1.0226630380829809
-  - 0.9624090336003619
-  - 1.355921271936585
-  - 0.7956901308566088
-  - 1.099707783022172
-  - 1.0931579604761017
+    [ 1.0609177678010249,          1.206946443499336,           1.355948505402434,
+      0.7227521436404027,          0.7575984344708557,          0.7956741498662401,
+      0.1957365647012886,          0.20799163604479487,         0.22190873172798262,
+      0.8535584007953777,          0.962304794061175,           1.0930365493770196   ]
   covariance:
-    - [ 3.74792330e-03, 5.36439155e-04, 1.54511969e-04, 1.24082050e-03, 2.54790295e-03, 1.96662980e-04, -8.47176807e-06, 6.89539832e-04, 2.05822396e-03, 6.69101857e-05, -2.00545941e-04, 4.95392172e-04]
-    - [0.00053644, 0.00049785, 0.00044108, 0.00069925, 0.00028, 0.00026589, 0.00026333, 0.00037546, 0.0001216, 0.00018052, 0.00015032, 0.00025083]
-    - [ 1.54511969e-04, 4.41081718e-04, 4.57952012e-03, -1.07231412e-03, -7.13130786e-06, 1.97444377e-04, 3.41960171e-03, -6.48331701e-04, -1.86872088e-04, 8.32354293e-05, 2.95541036e-03, -5.05556677e-04]
-    - [ 0.00124082, 0.00069925, -0.00107231, 0.00192956, 0.00078078, 0.00038531, -0.00084086, 0.00111799, 0.00052579, 0.00027652, -0.00089323, 0.00084393]
-    - [ 2.54790295e-03, 2.79998312e-04, -7.13130786e-06, 7.80783394e-04, 2.48575036e-03, 1.41768978e-04, -1.21693999e-04, 6.10240590e-04, 2.34821514e-03, 7.75276108e-05, -2.19894476e-04, 5.42070564e-04]
-    - [1.96662980e-04, 2.65888767e-04, 1.97444377e-04, 3.85310384e-04, 1.41768978e-04, 2.29504620e-04, 1.66868207e-04, 3.37106630e-04, 9.90275544e-05, 2.05523524e-04, 1.39534791e-04, 3.02284982e-04]
-    - [-8.47176807e-06, 2.63329835e-04, 3.41960171e-03, -8.40856127e-04, -1.21693999e-04, 1.66868207e-04, 3.35235988e-03, -7.44528584e-04, -2.20299575e-04, 9.61170483e-05, 3.26964526e-03, -6.23823284e-04]
-    - [ 0.00068954, 0.00037546, -0.00064833, 0.00111799, 0.00061024, 0.00033711, -0.00074453, 0.00106865, 0.00054207, 0.0003157, -0.00080303, 0.00101311]
-    - [ 2.05822396e-03, 1.21604831e-04, -1.86872088e-04, 5.25790127e-04, 2.34821514e-03, 9.90275544e-05, -2.20299575e-04, 5.42070896e-04, 2.55858068e-03, 8.88370565e-05, -2.67238657e-04, 5.89567919e-04]
-    - [6.69101857e-05, 1.80524096e-04, 8.32354293e-05, 2.76516725e-04, 7.75276108e-05, 2.05523524e-04, 9.61170483e-05, 3.15700426e-04, 8.88370565e-05, 2.18889248e-04, 1.08847797e-04, 3.33631399e-04]
-    - [-0.00020055, 0.00015032, 0.00295541, -0.00089323, -0.00021989, 0.00013953, 0.00326965, -0.00080303, -0.00026724, 0.00010885, 0.00357825, -0.00070352]
-    - [ 0.00049539, 0.00025083, -0.00050556, 0.00084393, 0.00054207, 0.00030228, -0.00062382, 0.00101311, 0.00058957, 0.00033363, -0.00070352, 0.00111506]
+    - - 0.003748073857835731
```

Comment:
Can you keep the old format for the covariance, it makes debugging and reading much simpler